stateproof/builder.go

```diff
@@ -402,7 +403,14 @@ func (spw *Worker) tryBroadcast() {
 	spw.mu.Lock()
 	defer spw.mu.Unlock()
 
-	for rnd, b := range spw.builders {
+	rounds := make([]basics.Round, 0, len(spw.builders))
+	for rnd := range spw.builders {
+		rounds = append(rounds, rnd)
+	}
+	sort.Slice(rounds, func(i, j int) bool { return rounds[i] < rounds[j] })
```

Comment:
why don't we export that into a function? and rename `rounds` to have an hint that it is ordered slice

Reply:
It's a one-line function call..